packages/components/src/components/editable/Cell.tsx

```diff
@@ -43,6 +43,8 @@ interface Props {
     selection?: boolean;
     values?: List<ValueDescriptor>;
     onCellModify?: () => any;
+    filteredLookupValues?: List<string>,
+    filteredLookupKeys?: List<any>,
```

Comment:
Would this be better as a (Ordered?)Map or a List of objects with `key` and `value` fields instead of two different properties?

Reply:
I think probably not. I can imagine in some cases user only have the list of keys or values, but not both.